### PR TITLE
Update user documentation

### PR DESCRIPTION
**Pull request by jamessspanggg**

### CODE REVIEW

**Comment by gohyinhao** [comment]
Made suggested changes for debt section. Also, suggested wording and formatting changes to ensure consistency throughout document

**Comment by jhocx** [comment]
Suggest including the following statement taken from the 'edit' feature:
"You can omit the (optional) parameters by leaving it empty. If all parameters are empty, no edits will occur.
and include it in 'editBudget' as well.

**Comment by gohyinhao** [comment]
Looks good!

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -123,50 +127,260 @@ Format 2: `edit INDEX`
...
+Examples:
+
+* `editBudget c/food $/500`
+* `editBudget c/all ed/31-3-2019`
+* `editBudget c/all $/5000 sd/1-1-2019 ed/31-3-2019`
+
+Format 2: `editBudget`
+
+Examples:
+
```
Comment (jhocx) [inline docs/UserGuide.adoc]: Suggest to add the following line from 'edit' into 'editBudget' as it is applicable as well. 

"You can omit the (optional) parameters by leaving it empty. If all parameters are empty, no edits will occur."

**Comment by jhocx** [comment]
Changes are required to 'undo' and 'redo' features, however I cannot make a code suggestion on those lines of code in this Pull Request. 

Will make a new PR with changes to 'undo' and 'redo' after this is approved and merged.

**Pull request merged**